include/deal.II/fe/fe_interface_values.h

```diff
@@ -705,6 +746,95 @@ FEInterfaceValues<dim, spacedim>::average(
   return value;
 }
 
+
+
+template <int dim, int spacedim>
+Tensor<1, dim>
+FEInterfaceValues<dim, spacedim>::average_gradient(
+  const unsigned int interface_dof_index,
+  const unsigned int q_point,
+  const unsigned int component) const
+{
+  const auto dofpair = dofmap[interface_dof_index];
```

Comment:
Our usual naming convention would suggest this:
```suggestion
  const auto dof_pair = dofmap[interface_dof_index];
```
(I guess it would also suggest `dof_map`, but you're not defining this here, so let's leave it for a later patch.)